src/App.css

```diff
@@ -1,32 +1,92 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
 .App {
-  text-align: center;
+  margin: 0;
+  padding: 0;
+  background-color: #050417;
+  width: 100vw;
+  height: 100vh;
+  color: #ffffff;
+  display: flex;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
+.Header {
+  background-color: #050417;
+  color: #ffffff;
+  margin: auto;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.Logo-wrapper {
+  width: 213px;
+  height: 120px;
+  margin: auto auto;
+}
+
+.Logo-rectangle {
+  margin: auto;
+  width: 70px;
+  height: 70px;
+  background: linear-gradient(
+    333.23deg,
+    #2a2b8d -16.84%,
+    #2b2f8f -12.31%,
+    #559bd1 119.72%
+  );
+  background-size: cover;
+  border-radius: 20px;
+  text-align: center;
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+.Logo {
+  display: block;
+  margin: auto;
+  width: 40px;
+  height: 40px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.Logo-wrapper > h1 {
+  font-family: OpenSans;
+  font-size: 36px;
+  text-align: center;
 }
+
+.Content {
+  display: none;
+}
+input {
+  width: 400px;
+  height: 50px;
+  border-radius: 4px;
+}
+.dropdown {
+  width: 400px;
+  height: 50px;
+  margin-top: 2vh;
```

Comment:
Please use `px` unit for margin. Also you can use just one CSS property for defining both `margin-top` and `margin-bottom` in one line